### PR TITLE
Add plain text version of Devise emails #516

### PR DESCRIPTION
**Pull request by jamgar**
Add plain text version of Devise emails for Issue #516

https://github.com/joemasilotti/railsdevs.com/issues/516

## Pull request checklist

- [ ] My code contains tests covering the code I modified - Did not add test code as these were text files, but did manually test sending the emails and reviewing them with letter_opener
- [x] I linted and tested the project with `bin/check`
- [x] I added significant changes and product updates to the [changelog](CHANGELOG.md)

### CODE REVIEW

File: app/views/devise/mailer/email_changed.text.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Hello <%= @email %>!
+
+<% if @resource.try(:unconfirmed_email?) %>
+  We're contacting you to notify you that your email is being changed to <%= @resource.unconfirmed_email %>.
```
Comment (joemasilotti) [inline app/views/devise/mailer/email_changed.text.erb:4]:
```suggestion
We're contacting you to notify you that your email is being changed to <%= @resource.unconfirmed_email %>.
```

File: app/views/devise/mailer/email_changed.text.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+<% else %>
+  We're contacting you to notify you that your email has been changed to <%= @resource.email %>.
```
Comment (joemasilotti) [inline app/views/devise/mailer/email_changed.text.erb:6]:
```suggestion
We're contacting you to notify you that your email has been changed to <%= @resource.email %>.
```

File: app/views/devise/mailer/password_change.text.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Hello <%= @resource.email %>!
+
+We're contacting you to notify you that your password has been changed.
+
```
Comment (joemasilotti) [inline app/views/devise/mailer/password_change.text.erb:4]:
```suggestion
```

File: app/views/devise/mailer/unlock_instructions.html.erb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @resource.email %>!</p>
```
Comment (joemasilotti) [inline app/views/devise/mailer/unlock_instructions.html.erb:1]: railsdevs doesn't use locking, so we can delete this file along with the text version.

**Review by joemasilotti — CHANGES_REQUESTED** [review]
Looking good! A few small code changes which you should be able to commit directly on GitHub. And the CHANGELOG.

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,78 +4,79 @@
 
 ### June
```
[the remark this answers is not in the document]
Comment (jamgar) [reply CHANGELOG.md:5]: Will this require needing to running git revert on my local machine?

[the remark this answers is not in the document]
Comment (joemasilotti) [reply CHANGELOG.md:5]: You could revert it and reapply the change or manually update the file. Also, happy to fix this one for you if you'd like!

File: app/views/devise/mailer/password_change.text.erb (hunk 3 above)
Comment (joemasilotti) [reply app/views/devise/mailer/password_change.text.erb:4]: Nope, just apply the diff I commented on. The indentation in text emails causes it to be indented when rendered.

File: CHANGELOG.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jamgar) [reply CHANGELOG.md:5]: I had a formatter enable in my editor that changed the asterisk to a dash 😞 Disable it and fixed it.

**Pull request merged**